README.md

```diff
@@ -544,7 +544,8 @@ Note: `petName` is only included if the game sent it to your chat via untradeabl
   "extra": {
     "questName": "Cook's Assistant",
     "personalBest": "1:13.20",
-    "currentTime": "1:13.20"
+    "currentTime": "1:13.20",
+    "isPersonalBest": true
```

Comment:
is this a correct example?

Reply:
I guess it can be read both ways, depending on if you interpret `personalBest` being the one before this attempt or not

Reply:
It's correct since we don't keep a record of previous pb's so only have the widget data to go off of